src/common/segment-utils.ts

```diff
@@ -104,3 +105,63 @@ type FacebookBasicEvent =
 export const trackFacebookBasicEvent = (eventName: FacebookBasicEvent) => {
   analytics.track(eventName, {}, { integrations: { All: false, 'Facebook Pixel': true } });
 };
+
+/**
+ * Register a Segment plugin that will parse url paths from learn.xip.co.
+ *
+ * @see https://github.com/segmentio/analytics-next/tree/master/packages/browser#-plugins
+ *
+ */
+export const urlPathFilter: Plugin = {
+  name: 'Clickfunnels Path Filter',
+  type: 'enrichment',
+  version: '1.0.0',
+
+  isLoaded: () => true,
+  load: () => Promise.resolve(),
+
+  track: (ctx) => {
+    const url = new URL(ctx.event.properties?.url || '');
+    if (url.hostname !== 'learn.xip.co') {
+      return ctx;
+    }
+
+    let urlPath = url.pathname;
+
+    // Handle the case where there is an extra '-page' at the end of the path
+    if (urlPath.endsWith('-page')) {
+      urlPath = urlPath.slice(0, -5);
+    }
```

Comment:
Would this be the only case? What does -page in the path refer to here?

Reply:
updated with an explanation. Clickfunnels has a page and a funnel page in short. I wanted to make sure we handle both cases.